commit 09795c5612c630db605886dfd55dbf56f381d128
Author: Wilco Dijkstra <wilco.dijkstra@arm.com>
Date:   Fri Jun 6 13:15:30 2025 +0000

    AArch64: Fix builderror with GCC 12.1/12.2
    
    Early versions of GCC 12 didn't support -mtune=neoverse-v2, so use
    -mtune=neoverse-v1 instead.
    
    Reported-by: Yury Khrustalev <yury.khrustalev@arm.com>

diff --git a/sysdeps/aarch64/fpu/Makefile b/sysdeps/aarch64/fpu/Makefile
index aa547b21df5f41d9..c8a6fb4628d13aec 100644
--- a/sysdeps/aarch64/fpu/Makefile
+++ b/sysdeps/aarch64/fpu/Makefile
@@ -53,7 +53,7 @@ endif
 # add a generic -mcpu and -march with SVE enabled.  Also use a tune for a modern
 # SVE core.
 
-sve-cflags = -mcpu=generic+sve -march=armv8-a+sve -mtune=neoverse-v2
+sve-cflags = -mcpu=generic+sve -march=armv8-a+sve -mtune=neoverse-v1
 
 ifeq ($(build-mathvec),yes)
 bench-libmvec = $(addprefix float-advsimd-,$(float-advsimd-funcs)) \
